config/v1/types_authentication.go

```diff
@@ -284,10 +284,10 @@ type TokenIssuer struct {
 }
 
 type TokenClaimMappings struct {
-	// username is an optional field that configures how the username of a cluster identity
+	// username is a required field that configures how the username of a cluster identity
 	// should be constructed from the claims in a JWT token issued by the identity provider.
 	//
-	// +optional
+	// +required
 	Username UsernameClaimMapping `json:"username,omitempty"`
```

Comment:
While we are here, this shouldn't have `omitempty` either

Reply:
dropped the `omitempty` in the latest push